ci/define-example-runner.sh

```diff
@@ -191,3 +191,28 @@ run_example_usage() {
   set -e
   /bin/rm -f "${log}"
 }
+
+################################################
+# Report the result of running many examples.
+# Globals:
+#   COLOR_*: colorize output messages, defined in colors.sh
+#   EXIT_STATUS: control the final exit status for the program.
+#   EMULATOR_LOG: the name of the emulator logfile.
+# Arguments:
+#   None
+# Returns:
+#   None
+################################################
+exit_example_runner() {
+  if [[ "${EXIT_STATUS}" == 0 ]]; then
+    echo
+    echo "${COLOR_GREEN}All the examples finished successfully.${COLOR_RESET}"
+    echo
+  else
+    echo
+    echo "${COLOR_RED}Some of the examples failed.${COLOR_RESET}"
+    echo
+  fi
+
+  exit "${EXIT_STATUS}"
+}
```

Comment:
I think you can do I think "trap exit_example_runner EXIT" here to have it auto-execute on exit?

Reply:
Unfortunately there is already a trap present when using the testbench or the emulator.  And traps do not stack.

Reply:
ok... you can use 'trap -p' to get the current trap and chain them together, but of course that assumes everyone cooperates or it's subject to ordering...  so let's leave this for now.